STEERING-COMMITTEE.md

```diff
@@ -162,13 +118,158 @@ There are two ways to raise issues to the steering committee for decision:
 2. Open an issue on a knative repository and indicate that you would like
    attention from the steering committee.
 
+## Composition
+
+The steering committee has 7 seats.  Five of the seats will be Contributing
+seats, open to any project contributor.  Two of the seats will be End-User
+seats, open only to active end-user community members.  See Candidate
+Eligibility for definitions of these categories.
+
+Steering Committee members serve for 2-year terms, staggered in order to
+preserve continuity.  Every year either two or three contributor seats are
+elected, and one end user seat is elected.
+
+## Election Procedure
+
+### Timeline
+
+Steering Committee elections are held annually. Six weeks or more before the
+election, the Steering Committee will appoint Election Officer(s) (see below).
+Four weeks or more before the election, the Election Officer(s) will issue a
+call for nominations, publish the list of voters, and open the call for
+exceptions.  One week before the election the call for nominations and
+exceptions will be closed.  The election will be open for voting not less than
+two weeks and not more than four.  The results of the election will be
+announced within one week of closing the election.  New Steering Committee
+members will take office on December 1 of each year.
+
+### Election Officer(s)
+
+Six weeks or more before the election, the Steering Committee will appoint
+between one and three Election Officer(s) to administer the election.  Elections
+Officers will be Knative community members in good standing who are eligible to
+vote, are not running for Steering in that election, and can make a public
```

Comment:
It's not clear to me from this paragraph if standing Steering Committee members may also be Election Officers

Reply:
Good point, I will incorporate this explicitly